doc/migration-to-tt.rst

```diff
@@ -0,0 +1,52 @@
+Migration from Cartridge CLI to tt
+==================================
+
+To switch from Cartridge CLI to ``tt``
```

Comment:
I think it's better to leave a link to https://github.com/tarantool/tt/blob/master/doc/examples.md#transition-from-cartridge-cli-to-tt here.

Reply:
Do you mean instead of this instruction or in addition?
We should have migration instructions published in Tarantool docs, so I wouldn't leave **only** the link to GitHub.